fe/fe-core/src/main/java/org/apache/doris/planner/SortNode.java

```diff
@@ -211,9 +211,6 @@ public void init(Analyzer analyzer) throws UserException {
         outputSmap = new ExprSubstitutionMap();
 
         for (int i = 0; i < slotExprs.size(); ++i) {
-            if (!sortTupleSlots.get(i).isMaterialized()) {
```

Comment:
add a comment that we rely on projectionPlanner to get a correct plan.

Reply:
done